R/module_teal.R

```diff
@@ -121,7 +123,7 @@ ui_teal <- function(id,
 
 
 #' @rdname module_teal
-srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
+srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices(), lockfile_task) {
```

Comment:
if addition of a new argument is unavoidable - can we please make it optional and then process it if it's present (not null)? This is all to make clear distinguish between what's important and what's not.